webui/react/src/pages/ResourcePool/ClusterTopology.tsx

```diff
@@ -0,0 +1,73 @@
+import Tooltip from 'hew/Tooltip';
+import React, { PropsWithChildren, useEffect, useMemo, useRef, useState } from 'react';
+
+import Section from 'components/Section';
+import { Agent, Resource, SlotsRecord } from 'types';
+
+import css from './Topology.module.scss';
```

Comment:
We need to restore the previous css file, too

Reply:
oops, thanks